omero/sysadmins/unix/install-web/install-apache.txt

```diff
@@ -12,6 +12,9 @@ Apache 2.2+ with mod_wsgi configuration (Unix/Linux)
       `Django 1.8`_ (LTS) which requires Python 2.7. For more information
       see :ref:`python-requirements` on the
       :doc:`/sysadmins/version-requirements` page.
+      
+      Also note that Apache is deprecated as of OMERO 5.2.6 and will not be
+      supported for 5.3.
 
```

Comment:
I think this is not well visible, also would be possible to make a clearly visible note in a upgrade section somewhere in http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/server-upgrade.html#omero-web-configuration